src/rpm/package.rs

```diff
@@ -335,6 +323,12 @@ impl RPMPackageMetadata {
             .get_entry_data_as_string_array(IndexTag::RPMTAG_FILEDIGESTS)
     }
 
+    #[inline]
+    pub fn get_file_ima_signatures(&self) -> Result<&[String], RPMError> {
+        self.signature
+            .get_entry_data_as_string_array(IndexSignatureTag::RPMSIGTAG_FILESIGNATURES)
+    }
```

Comment:
This is just moved, if you look at the functions above and below it makes more sense down here.